turbo.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "globalEnv": ["GITHUB_TOKEN"],
+  "tasks": {
+    "build": {
+      "dependsOn": ["^build"]
```

Comment:
Recommended to read the [Turborepo docs](https://turbo.build/repo/docs/reference/configuration), but the `^` character means “it has no dependencies.” Why they did this, and not an empty array, I do not know. But that’s the way it is.